src/components/TodoItem.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/state-in-constructor */
```

Comment:
Please kindly  disable `eslint` on your codebase. `eslint` enables you to go through the best practices on javascript code. refer [wikipedia](https://en.wikipedia.org/wiki/ESLint) 👍